sup3r/models/base.py

```diff
@@ -350,8 +350,8 @@ def set_model_params(self, **kwargs):
                 msg = ('GAN was previously trained with {var}={} but '
                        'received new {var}={}'
                        .format(val, kwargs[var], var=var))
-                logger.error(msg)
-                raise KeyError(msg)
+                logger.warning(msg)
```

Comment:
yeah good call. had to edit the json for some things weve updated

Reply:
yeah totally. we should give ourselves the power to manipulate these models in crazy ways muahhaha